assets-src/js/multiselect-module.js

```diff
@@ -203,13 +215,12 @@ const MultiselectButtons = function (selectEl, params) {
             buttonEl.className = 'remove-option';
             buttonEl.type = 'button';
             buttonEl.id = `${this.idBase}-remove-${index}`;
-            buttonEl.setAttribute('aria-describedby', `${this.idBase}-remove`);
             buttonEl.dataset.value = option.value;
             buttonEl.addEventListener('click', () => {
                 this.removeOption(option);
             });
 
-            buttonEl.innerHTML = option.text + ' ';
+            buttonEl.innerHTML = `<span class="sr-only">Remove </span>${option.text} `;
```

Comment:
I believe this class is for bootstrap. You should probably use .visuallyhidden instead (see https://design-system.w3.org/styles/how-to-hide-and-show-things.html#visually-hiding-items)

Reply:
Addressed by @deniak in https://github.com/w3c/w3c-website-templates-bundle/commit/6f17ef0ac7a0535df34bd9a91876eecb70a1290c